### PR TITLE
LRQA-57134 Add a new test to check refresh token's lifetime variable operates as expected

### PR DESCRIPTION
**Pull request by gaborlovasdotcom**
LRQA-57134 New test added

### CODE REVIEW

**Comment by gaborlovasdotcom** [comment]
Sent for review @xbrianlee , @manoelcyreno.

**Comment by manoelcyreno** [comment]
can you rename your commit to use something more clear?
example: "LRQA-57134 Add a new test to refresh token's lifetime"...

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase [hunk 1]
```diff
@@ -603,6 +603,94 @@ definition {
...
+
+		Button.click(button = "Obtain OAuth2 Access");
+
+		Button.click(button = "Launch Request");
+
+		AssertTextEquals.assertPartialText(
+			locator1 = "OAuth2#OAUTH2_TESTER_REQUEST_RESULTS",
+			value1 = "test@liferay.com");
+
+		Pause(locator1 = "6000");
```
Comment (manoelcyreno) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase:675]: Why you have this pause here?
You can not replace it by some wait?

Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase:675]: I asked what type of pause methods we have, and Carlos B. mentioned we have this format.

**Pull request closed without merging**